plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/intermediate/IntermediateProcessorStrategy.java

```diff
@@ -42,7 +42,7 @@
     EDGE_AND_LAYER_CONSTRAINT_EDGE_REVERSER,
     /** If one of the phases is set to interactive mode, this processor positions external ports. */
     INTERACTIVE_EXTERNAL_PORT_POSITIONER,
-    /** Add constraint edges to respect partitioning of nodes. */
+    /** Reverse edges that run contrary to layout partitions. */
```

Comment:
I believe _contrary to_ is not the right expression here. Maybe "that connect partitions of higher index with partitions of lower index"?

(Same in the class's javadoc).